client/src/main/scala/drt/client/components/TerminalComponent.scala

```diff
@@ -94,6 +95,10 @@ object TerminalComponent {
             val staffingContentClass = if (props.terminalPageTab.mode == "staffing") "fade in active" else "fade out"
             val dashboardContentClass = if (props.terminalPageTab.mode == "dashboard") "fade in active" else "fade out"
 
+            val coachWalkTime = airportConfig.portCode match {
```

Comment:
As this block of code is repeated in 3 places, we could move it to an `apply` method on a new `CoachWalkTime` object which just takes the `PortCode`